cmake/modules/extensions.cmake

```diff
@@ -3332,6 +3332,49 @@ macro(zephyr_linker_memory_ifdef feature_toggle)
   endif()
 endmacro()
 
+# Usage:
+#   zephyr_linker_dts_section_from_path(PATH <path>)
```

Comment:
minor proposal.

We already specify `PATH` in the argument list, so why have it twice like this: `___from_path(PATH ...)`, why not simply call it:
```
zephyr_linker_dts_section(PATH <path>)
```
which also aligns in naming with the memory function below:
```
zephyr_linker_dts_memory(PATH <path> ...)
```
https://github.com/zephyrproject-rtos/zephyr/blob/eee0da7fc1d79ecc29657ec37c1c310ac56a1053/cmake/modules/extensions.cmake#L3379

Reply:
makes sense.